devops/automation/azure-pipelines.yml

```
@@ -116,29 +116,29 @@ jobs:
     displayName: 'Clean workspace'
     workingDirectory: $(Build.SourcesDirectory)
 
-  - script: |
-      set -e
-      set -x
-      $(Build.SourcesDirectory)/binding-tools-for-swift/devops/automation/build-swift.sh --publish
-      mkdir -p $(System.DefaultWorkingDirectory)/toolchain
-      if test -f $(System.DefaultWorkingDirectory)/binding-tools-for-swift/Pack-Man/SwiftToolchain-*.zip; then
-        cp -c $(System.DefaultWorkingDirectory)/binding-tools-for-swift/Pack-Man/SwiftToolchain-*.zip $(System.DefaultWorkingDirectory)/toolchain
-        echo "##vso[task.setvariable variable=uploadtoolchain;isOutput=true]1"
-      else
-        echo "##vso[task.setvariable variable=uploadtoolchain;isOutput=true]0"
-      fi
-    displayName: 'Build Swift'
-    name: BuildSwift
+#  - script: |
+#      set -e
+#      set -x
+#      $(Build.SourcesDirectory)/binding-tools-for-swift/devops/automation/build-swift.sh --publish
+#      mkdir -p $(System.DefaultWorkingDirectory)/toolchain
+#      if test -f $(System.DefaultWorkingDirectory)/binding-tools-for-swift/Pack-Man/SwiftToolchain-*.zip; then
+#        cp -c $(System.DefaultWorkingDirectory)/binding-tools-for-swift/Pack-Man/SwiftToolchain-*.zip $(System.DefaultWorkingDirectory)/toolchain
+#        echo "##vso[task.setvariable variable=uploadtoolchain;isOutput=true]1"
+#      else
+#        echo "##vso[task.setvariable variable=uploadtoolchain;isOutput=true]0"
+#      fi
+#    displayName: 'Build Swift'
+#    name: BuildSwift
```

instead of commenting out, you can just disable the step with:
```yaml
enabled: boolean 
```
ref: https://docs.microsoft.com/en-us/azure/devops/pipelines/process/tasks?view=azure-devops&tabs=yaml

Proposed change:
```suggestion
  - script: |
      set -e
      set -x
      $(Build.SourcesDirectory)/binding-tools-for-swift/devops/automation/build-swift.sh --publish
      mkdir -p $(System.DefaultWorkingDirectory)/toolchain
      if test -f $(System.DefaultWorkingDirectory)/binding-tools-for-swift/Pack-Man/SwiftToolchain-*.zip; then
        cp -c $(System.DefaultWorkingDirectory)/binding-tools-for-swift/Pack-Man/SwiftToolchain-*.zip $(System.DefaultWorkingDirectory)/toolchain
        echo "##vso[task.setvariable variable=uploadtoolchain;isOutput=true]1"
      else
        echo "##vso[task.setvariable variable=uploadtoolchain;isOutput=true]0"
      fi
    displayName: 'Build Swift'
    name: BuildSwift
    enabled: false
```